### DISCUSSION TITLE
How to deploying nats in Azure Kubernetes Service (AKS) through terraform

### DISCUSSION THREAD

**Discussion by bobbybhaskar** [nats-io/k8s#439] [Q&A]
How to create or deploying nats on Azure Kubernetes Service (AKS) by using terraform deployment script? I have create the other "resource". Is there any other smart way of deploying nats for aks through [terraform] script.

**Comment by wallyqs** [comment]
Hi @bobbybhaskar, you can find some terraform related tools for JetStream here: https://github.com/nats-io/terraform-provider-jetstream
